app/pathogen/arbovirus/wip/sections.tsx

```diff
@@ -0,0 +1,73 @@
+import { RefObject } from "react";
+
+enum RedesignedArbovirusPageSectionId {
+  MAP = 'MAP',
+  TABLE = 'TABLE',
+  VISUALIZATIONS = 'VISUALIZATIONS'
+}
+
+enum RedesignedArbovirusPageSectionUrlParam {
+  map = 'map',
+  table = 'table',
+  visualizations = 'visualizations'
+}
+
+const isRedesignedArbovirusPageSectionUrlParam = (urlParam: string): urlParam is RedesignedArbovirusPageSectionUrlParam => {
+  return Object.values(RedesignedArbovirusPageSectionUrlParam).some((element) => element === urlParam);
+}
+
+export const redesignedArbovirusPageSections = [{
+  id: RedesignedArbovirusPageSectionId.MAP,
+  url: RedesignedArbovirusPageSectionUrlParam.map,
+  renderScrollSectionContent: (input: { ref: RefObject<HTMLElement>, key: string}) => (
+    <section
+      key={input.key}
+      ref={input.ref}
+      className="w-full h-[95%] bg-blue-500 snap-start snap-always scroll-smooth"
```

Comment:
What I love so much about the `h-[95%]` is that if it's not the last section, the next section peeks up below the page a little bit to let you know that there's another section and you can scroll down. I think this will look really nice hopefully once the sections aren't just coloured rectangles lol

![Peek 2024-01-21 16-23](https://github.com/PathoTracker/Pathotracker/assets/86806388/70bb22cb-cc93-4338-9564-ec152ca99fe9)